extensions-contrib/influxdb-emitter/src/main/java/org/apache/druid/emitter/influxdb/InfluxdbEmitterConfig.java

```diff
@@ -130,6 +159,10 @@ public int hashCode()
   {
     int result = getHostname().hashCode();
     result = 31 * result + getPort();
+    result = 31 * result + getProtocol().hashCode();
+    result = 31 * result + (getTrustStorePath() != null ? getTrustStorePath().hashCode() : 0);
+    result = 31 * result + (getTrustStoreType() != null ? getTrustStoreType().hashCode() : 0);
+    result = 31 * result + (getTrustStorePassword() != null ? getTrustStorePassword().hashCode() : 0);
```

Comment:
Looks like CI is complaining about a lack of test coverage. You can add an `EqualsVerifier` test to fix that

Reply:
How does `EqualsVerifier` work? The test I added is failing on 

> Non-nullity: equals throws NullPointerException on field port.

but if port is null then it gets set to a default value in the config's constructor.

Reply:
EqualsVerifier doesn't know that because that field is not marked as `NotNull`. There's more information about that exception here - https://jqno.nl/equalsverifier/errormessages/non-nullity-equals-hashcode-tostring-throws-nullpointerexception/

You could also tell the test that the field is not null with `.withNonnullFields("port")` in the EqualsVerifier test